internals/feature_helpers.py

```diff
@@ -85,7 +85,10 @@ def get_features_in_release_notes(milestone: int):
   features = [dict(converters.feature_entry_to_json_verbose(f))
             for f in _get_future_results(_get_entries_by_id_async(feature_ids))]
   features = [f for f in filter_unlisted(features) 
-    if f['deleted'] == False and (f['breaking_change'] == True or f['feature_type_int'] == FEATURE_TYPE_ENTERPRISE_ID)]
+    if f['deleted'] == False and 
```

Comment:
This code could be a little cleaner if we don't compare to boolean constants.  I.e.,
`if not f['deleted'] and ... `